cypress/integration/governanceReviewTeam.spec.js

```diff
@@ -98,4 +98,70 @@ describe('Governance Review Team', () => {
       cy.get('[data-testid="user-note"]').first().contains('User GRTB');
     });
   });
+
+  it('can issue a Lifecycle ID', () => {
+    // Selecting name based on pre-seeded data
+    // A Completed Intake Form - af7a3924-3ff7-48ec-8a54-b8b4bc95610b
+    cy.get('a').contains('A Completed Intake Form').click();
+    cy.get(
+      'a[href="/governance-review-team/af7a3924-3ff7-48ec-8a54-b8b4bc95610b/actions"]'
+    )
+      .contains('Take an action')
```

Comment:
Test is failing here for me.

Reply:
hmm, not for me?

Reply:
There's a race condition from when the "loading" is rendered and when  the page is fully loaded.

If the page is still loading, it will find the "Action" link in the side bar instead of the link at the bottom of the fully rendered page.